src/main/java/ch/njol/skript/classes/Cloner.java

```diff
@@ -3,7 +3,9 @@
 /**
  * An interface for optionally cloning an object,
  * should return the given object if no cloning is required.
+ * @deprecated Use a {@code Function<T, T>}.
  */
+@Deprecated(forRemoval = true)
 public interface Cloner<T> {
 	
 	T clone(T t);
```

Comment:
Why is this being deprecated? It's nice to have more semantic meaning in interfaces. Just make it a FunctionalInterface.